apps/mudita-center-e2e/src/consts/mockresponse-const.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+export const mockResponseTomasz = {
```

Comment:
Could we reword this variable to better describe its use case from a technical perspective? 🙏

Reply:
IMO it should be removed, I left it for testing purposes and I will removed before merge

Reply:
We will revert changes before merging

Reply:
Cleaned